### ISSUE TITLE
Request for a ML/Google group

### ISSUE THREAD

**Issue by NinjaComics** [ptrkrysik/gr-gsm#76]
@ptrkrysik Now that gr-gsm is gaining some momentum, can I suggest that we have a mailing list/google group where issues like #75 and problems concerning installation can be tackled on the list and have gr-gsm-development only discussions on Github ? 

P.S - I haven't contributed a single line of code to gr-gsm, but I am following it and have used it previously. That said, I am looking forward to contributing a bit from my end as well.

**Comment by akibsayyed** [comment]
I can provide hosting for mailing list

On Thu, Jun 25, 2015 at 6:38 PM, Ravi Sharan <notifications@github.com>
wrote:

> @ptrkrysik <https://github.com/ptrkrysik> Now that gr-gsm is gaining some
> momentum, can I suggest that we have a mailing list/google group where
> issues like #75 <https://github.com/ptrkrysik/gr-gsm/issues/75> and
> problems concerning installation can be tackled on the list and have
> gr-gsm-development only discussions on Github ?
>
> P.S - I haven't contributed a single line of code to gr-gsm, but I am
> following it and have used it previously. That said, I am looking forward
> to contributing a bit from my end as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ptrkrysik/gr-gsm/issues/76>.
>



-- 
Akib Sayyed
Matrix-Shell
akibsayyed@gmail.com
akibsayyed@matrixshell.com
Mob:- +91-966-514-2243

**Comment by ptrkrysik** [comment]
I don't have too much experience in setting up and administering mailing list. I agree that there is need for place for helping new users and general discussions so I decided to start google group, as it can be configured immediately. This group is:

gr-gsm@googlegroups.com

I've set up the list with most liberal settings I could set. To join the group with any e-mail addres (google account is not required) use this link:
https://groups.google.com/forum/#!forum/gr-gsm/join
